lib/kernel/src/inet_res.erl

```diff
@@ -313,6 +320,8 @@ make_options([{Opt,Val}|Opts]=Opts0, [Opt|Names]=Names0) ->
     end;
 make_options(Opts, [verbose|Names]) ->
     [false|make_options(Opts, Names)];
+make_options(Opts, [nxdomain_reply|Names]) ->
```

Comment:
Would it be slightly more beautiful to have these clauses in sorted order too, i.e `nxdomain_reply` before `verbose`?